docs/guides/model_selection.md

```diff
@@ -0,0 +1,266 @@
+# Model Selection Guide
+
+This guide describes how to select specific models to include in a SQLMesh plan, which can be useful when modifying a subset of the models in a SQLMesh project.
+
+## Background
+
+A SQLMesh [plan](../concepts/plans.md) automatically detects changes between the local version of a project and the version deployed in an environment. When applied, the plan backfills the directly modified models and their indirectly modified downstream children. This brings all model data into alignment with the local version of the project.
+
+In large SQLMesh projects, a single model change may impact many downstream models, such that evaluating it and its affected children takes a significant amount of time. In some situations, a user is blocked by the long run time and can accomplish their task without backfilling all changed models and affected children.
+
+SQLMesh model selection allows you to filter a plan's directly changed models to only the ones necessary to accomplish your task. This can be useful when you only need to inspect the results of some of the model changes you have made.
```

Comment:
```suggestion
SQLMesh model selection allows you to filter which direct model changes should be included into a plan. This can be useful when you only need to inspect the results of some of the model changes you have made.
```